### DISCUSSION TITLE
IIV on RUV

### DISCUSSION THREAD

**Discussion by EmanIbrahim137** [nlmixrdevelopment/nlmixr#616] [Q&A]
Hi, 

I was wondering about how to include interindividual variability IIV on the magnitude of RUV.

Since the below parameterization did not work

IPRED ~ add(add.sd*exp(eta.ruv))

thanks in advance,
Eman

**Comment by mattfidler** [comment]
Hi @EmanIbrahim137,

This is an experimental feature in `nlmixr2`

To add the parameterization you would use:

```r
...
addErr <- add.sd*exp(eta.ruv)
IPRED ~ add(addErr)
...
```
Currently this is only supported by `focei`

**Reply by mattfidler** [reply]
But it isn't supported in `nlmixr`

**Reply by EmanIbrahim137** [reply]
thanks!
